src/bots/ping/handler.js

```diff
@@ -0,0 +1,8 @@
+import client from '../../client';
+
+const handler = (_, msg) => {
+  msg.channel.send('Pong!');
+  client.logger.debug('ping reached');
```

Comment:
let's remove the log here since we have the channel message

Reply:
Actually, I take this back this is ok

Reply:
I did that at one point but decided against removing it because it's used in the unit test (just like how it's used in your pins unit test). Is there a better function to spy on than `client.logger.debug()`?